pyhap/accessory.py

```diff
@@ -487,12 +479,7 @@ def to_HAP(self):
 
         .. seealso:: Accessory.to_HAP
         """
-        hap_rep = [super().to_HAP()]
```

Comment:
Not sure if this is more readable or more efficient than the previous version.

Reply:
I'm not 100% percent sure either. Just though about that line while I was working on the other PR and it looked pretty cool, especially since we already use list comprehensions in other places (mainly for the `to_HAP` methods actually).

At least it isn't worse than the before IMO, and it cuts five lines.

Reply:
It's not worse, performance-wise I think. Just readability, but it's good enough, Merging